### ISSUE TITLE
Proposing a set of features: let's dream!

### ISSUE THREAD

**Issue by carlisia** [dvargas92495/roamjs-github#1]
From a thread on the romresearch Slack roam42 channel, @dvargas92495 said:

> I also think the extension sort of needs to be redesigned from scratch. It's pretty hard to use at the moment and would love there to be better two way syncing in order for me to even use it with my graph

I want to kick this off with my wish list of features. Please add to it so we can have a good idea of what a redesign could look like.

Note: this list does not account for what is available with the GH API. In other words, some of what is listed might not be possible, that's ok.

For my own user, list open:
- issues/PRs I'm assigned to
- issues/PRs I'm mentioned
- issues/PRs I'm participating
- PRs that requested my review

Choose a public repository, and list:
- open issues
- open PRs

**Comment by dvargasfuertes** [comment]
One thing I realized is that I don't necessarily want to _import_ github data but _reference_ it instead, similar to block references.

So I want to be able to do something like `{{github:dvargas92495/roamjs-github/issues/1}}` and it opens a component that gives me a view of the issue, allow me to interact with it etc.

I also want to be able to promote pages/blocks in my graph to being github issues/projects on github

**Comment by joelchan** [comment]
> I also want to be able to promote pages/blocks in my graph to being github issues/projects on github

Yeah I want this also! And, like you said, preserve the reference in the graph.

**Comment by carlisia** [comment]
Hey everyone, I will not be able to contribute with this feature. Leaving the issue open in case someone else comes around to tackle it, but also feel free to close it.
